src/board/dto/response-board.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { CreateBoardDto } from './create-board.dto';
+import { PartialType } from '@nestjs/mapped-types';
+import {
+    IsNotEmpty,
+    IsString,
+} from 'class-validator';
+
+export class ResponseBoardDto extends PartialType(CreateBoardDto) {
```

Comment:
I think response should normally contain all fields with rare exceptions. Maybe this DTO is redundant